static/app/styles/global.tsx

```diff
@@ -193,6 +193,11 @@ const styles = (theme: Theme, isDark: boolean) => css`
     }
   }
 
+  .ReactVirtualized__Grid:focus-visible,
```

Comment:
what do these 2 lines do?

Reply:
Is this going to affect our other virtualized grids in unwanted ways?

Reply:
@billyvg It'll also update the focus color of the other ones yeah. We use them a lot for the replay tabs, but `outline` is not a layout changing property, so it's only the colors that will change.